src/Compilers/Lua/Portable/Syntax/SyntaxKind.cs

```diff
@@ -496,6 +496,12 @@ public enum SyntaxKind : ushort
         StringLiteralToken = 1002,
         [Token]
         IdentifierToken = 1003,
+        [Token]
+        [ExtraCategories(SyntaxKindCategory.LiteralToken)]
+        [Property(SyntaxKindProperty.LiteralExpression, HashStringLiteralExpression)]
+        HashStringLiteralToken = 1004,
+
+        
 
         // Big gap 1003-2000 (insert new tokens with text here)
```

Comment:
Please update the comment to the next free code as 1003 is being used for the hash string token